redirects.yaml

```
@@ -315,3 +315,117 @@ redirects:
+    destination: https://dequeuniversity.com/rules/axe/4.4/bypass
+  - source: /duplicate-id-active/
+    destination: https://dequeuniversity.com/rules/axe/4.4/duplicate-id-active
+  - source: /heading-order/
+    destination: https://dequeuniversity.com/rules/axe/4.4/heading-order
+  - source: /tabindex/
+    destination: https://dequeuniversity.com/rules/axe/4.4/tabindex
+  - source: /aria-allowed-attr/
+    destination: https://dequeuniversity.com/rules/axe/4.4/aria-allowed-role
+  - source: /aria-hidden-body/
+    destination: https://dequeuniversity.com/rules/axe/4.4/aria-hidden-body
+  - source: /aria-hidden-focus/
+    destination: https://dequeuniversity.com/rules/axe/4.4/aria-hidden-focus
+  - source: /aria-required-attr/
+    destination: https://dequeuniversity.com/rules/axe/4.4/aria-required-attr
+  - source: /aria-required-children/
+    destination: https://dequeuniversity.com/rules/axe/4.4/aria-required-children
+  - source: /aria-required-parent/
+    destination: https://dequeuniversity.com/rules/axe/4.4/aria-required-parent
+  - source: /aria-roles/
+    destination: https://dequeuniversity.com/rules/axe/4.4/aria-roles
+  - source: /aria-valid-attr-value/
+    destination: https://dequeuniversity.com/rules/axe/4.4/aria-valid-attr-value
+  - source: /aria-valid-attr/
+    destination: https://dequeuniversity.com/rules/axe/4.4/aria-valid-attr
+  - source: /duplicate-id-aria/
+    destination: https://dequeuniversity.com/rules/axe/4.4/duplicate-id-aria
+  - source: /button-name/
+    destination: https://dequeuniversity.com/rules/axe/4.4/aria-command-name
+  - source: /document-title/
+    destination: https://dequeuniversity.com/rules/axe/4.4/document-title
+  - source: /form-field-multiple-labels/
+    destination: https://dequeuniversity.com/rules/axe/4.4/form-field-multiple-labels
+  - source: /frame-title/
+    destination: https://dequeuniversity.com/rules/axe/4.4/frame-title
+  - source: /image-alt/
+    destination: https://dequeuniversity.com/rules/axe/4.4/image-alt
+  - source: /input-image-alt/
+    destination: https://dequeuniversity.com/rules/axe/4.4/input-image-alt
+  - source: /label/
+    destination: https://dequeuniversity.com/rules/axe/4.4/label
+  - source: /link-name/
+    destination: https://dequeuniversity.com/rules/axe/4.4/link-name
+  - source: /object-alt/
+    destination: https://dequeuniversity.com/rules/axe/4.4/object-alt
+  - source: /color-contrast/
+    destination: https://dequeuniversity.com/rules/axe/4.4/color-contrast
+  - source: /definition-list/
+    destination: https://dequeuniversity.com/rules/axe/4.4/definition-list
+  - source: /dlitem/
+    destination: https://dequeuniversity.com/rules/axe/4.4/dlitem
+  - source: /list/
+    destination: https://dequeuniversity.com/rules/axe/4.4/list
+  - source: /listitem/
+    destination: https://dequeuniversity.com/rules/axe/4.4/listitem
+  # TODO - there doesn't seem to be a corresponding axe rule.
+  - source: /layout-table/
+    destination:
```

```suggestion
  - source: /layout-table/
    destination: https://dequeuniversity.com/rules/axe/3.5/layout-table
```
layout-table was removed in axe-core 4.0